axolotl-web/src/pages/ChatList.vue

```diff
@@ -141,7 +141,9 @@ export default {
 </script>
 
 <!-- Add "scoped" attribute to limit CSS to this component only -->
-<style>
+<style lang="scss">
+@import '../assets/style.scss';
```

Comment:
I'd love to enable [this](https://vue-loader.vuejs.org/guide/pre-processors.html#sharing-global-variables) to avoid having to add that import in each file, but I'm not comfortable with `package.json`. Mind helping me on that point?